### ISSUE TITLE
WordPress Lateral Menu - Collapse & uncollapse - Blocked

### ISSUE THREAD

**Issue by frederikcs74** [woocommerce/woocommerce#28218]
Hello !

WooCommerce Version 4.6.2 on 5.5.3–fr_FR.

The lateral menu is blocked. When I turn off WooCommerce Plugin (this also turns off WooCommerce Admin) the WP menu uncollapses.

As soon as I turn WooCommerce Plugin back on the WP lateral menu (navigation bar in English?) collapses and remains blocked. 

How do we solve this? 

Many thanks,
FCS

![Capture d’écran 2020-11-09 à 19 38 43](https://user-images.githubusercontent.com/74207212/98582833-c7bdbd80-22c3-11eb-8b9b-e35da66ff46b.png)
![Capture d’écran 2020-11-09 à 19 43 36](https://user-images.githubusercontent.com/74207212/98582914-e328c880-22c3-11eb-8716-e6c6fe15f1b3.png)

**Comment by juliaamosova** [comment]
Hi @frederikcs74,

Thank you for submitting the issue. However, I can’t reproduce it using the steps you provided. Everything is working as expected on my end. I tested using WooCommerce 4.7 + WordPress 5.5.3:

![](http://g.recordit.co/UOBTvTGcsV.gif)

Can you please test with all the plugins deactivated on your site except for WooCommerce - can you replicate the issue when you turn off WooCommerce with all other plugins deactivated as well?

**Comment by frederikcs74** [comment]
Hi Julia,

Thanks for getting back to us. 

I did what you suggested and deactivated all plugins on my site except for WooCommerce. The left WP navigation menu remains blocked. The button Collapse and Uncollapse isn’t responding.
As soon as I deactivate WooCommerce the menu uncollapses and the button Collapse and Uncollapse responds properly.

I found these two articles reporting the same issue: 
https://wordpress.org/support/topic/expand-collapse-button-of-dashboard-lhs-menu-not-reacting/
https://wordpress.org/support/topic/un-collapsing-the-admin-dashboard-menu/

The only thing that resolves the blocked menu (Expand/Collapse) is deactivating the WooCommerce plugin (which I need obviously).
We have this issue on three websites.

I would be greatly appreciated if you have any suggestions. 

All the best,

Frederik STEENBRINK | Encore Music
E frederik@frederiksteenbrink.com   
T 06 03 53 06 67

L’album Oh Là Là d’Isabelle Georges <https://bit.ly/OhLaLaIsabelleGeorges>
